vshard/storage/init.lua

```diff
@@ -1808,6 +1809,9 @@ local function bucket_test_gc(bids)
     local _bucket = box.space._bucket
     local ref, bucket, status
     for i, bid in ipairs(bids) do
+        if M.rebalancer_transfering_buckets[bid] then
```

Comment:
Nit: we check, whether the bucket is being transferred, inconsistently. In some cases it's `M.rebalancer_transfering_buckets` in other - `bucket_is_transfer_in_progress`. Maybe we should make `bucket_is_transfer_in_progress` to check `M.rebalancer_transfering_buckets` and use it everywhere.